api/src/team/application/api/models/user-teams-access.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @class
+ * @classdesc Model representing a user teams access.
+ */
+export class UserTeamsAccess {
```

Comment:
Remarque générale de nommage,  `UserTeamsAccess` me fait un peu trop penser à l'équipe « Accès », alors je suggère quelque chose d'autre mais il y a peut être mieux : 
```suggestion
export class UserTeamsInfo {
```

Reply:
Renommé 👌